tasks/skylab/TagSortBam.wdl

```diff
@@ -27,10 +28,24 @@ task CellSortBam {
   
   command {
     set -e
+    mkdir temp
+    gunzip -c ~{original_gtf} > annotation.gtf
+
+    TagSort --bam-input ~{bam_input} \
+    --gtf-file annotation.gtf \
+    --metric-output cell-metrics.csv  --compute-metric \
+    --metric-type cell \
+    --barcode-tag CB \
+    --umi-tag UB \
+    --gene-tag GX \
```

Comment:
The parameter_metadata section says it uses GE tag for gene IDs

Reply:
corrected.

Reply:
It looks like it still says it:
```parameter_meta {
    bam_input: "Input bam file containing reads marked with tags for cell barcodes (CB), molecule barcodes (UB) and gene ids (GE)"
    docker: "(optional) the docker image containing the runtime environment for this task"
    machine_mem_mb: "(optional) the amount of memory (MiB) to provision for this task"
    cpu: "(optional) the number of cpus to provision for this task"
    disk: "(optional) the amount of disk space (GiB) to provision for this task"
    preemptible: "(optional) if non-zero, request a pre-emptible instance and allow for this number of preemptions before running the task on a non preemptible machine"
  }```